src/main/java/com/andreamazzarella/contact_manager/Age.java

```diff
@@ -0,0 +1,36 @@
+package com.andreamazzarella.contact_manager;
+
+import java.util.regex.Pattern;
+
+public class Age {
+    private final String age;
+
+    public Age(String age) {
+        age = removeAllWhiteSpace(age);
+
+        if (isValidNumber(age)) {
```

Comment:
if it's not valid, I would probably rather see a runtime exception raised. I think returning 0 is potentially surprising.